shared/bash_remediation_functions/rhel7_handle_ntp_like_file.sh

```diff
@@ -0,0 +1,16 @@
+# Function ensures that the ntp/chrony config file contains valid server entries
+# $1: Path to the config file
+# $2: Comma-separated list of servers
+function rhel7_handle_ntp_like_file {
```

Comment:
The function name should be more specific to what it does. Like adding or enabling the list of servers in ntp like files.